content/docs/cert-csi/_index.md

```diff
@@ -1,6 +1,7 @@
 ---
```

Comment:
As of now, I don't see a problem with the 19 items. But, if we decide to move the items under Support, let Cert-CSI and Troubleshooting be in the main left menu since they are/will be frequently used.